### PR TITLE
Combine Cypress E2E Landing Page and Early Access Page Tests

### PR DESCRIPTION
**Pull request by martin-carmona**
## Description

This pull request combines the Cypress E2E tests from the Early Access Page to the Landing Page due to the following PR:
 - https://github.com/Linkta-org/landing-page/pull/48

Fixes #378

## Type of change

- [x] Other 🗒️ Update Cypress E2E Coverage

## How Has This Been Tested?

- [x] Verified that Cypress test succeeds locallly
- [x] Verified that Cypress GitHub Action succeeds

## Checklist:

- [x] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [x] I have reviewed the [**Security Best Practices Document**](SECURITY_BEST_PRACTICES.md).
- [x] I have followed the [**Naming Conventions Guide**](NAMING_CONVENTIONS_GUIDE.md) for my changes.
- [x] I have ensured that my pull request title is descriptive.

### CODE REVIEW

**Review by kinskiwu — APPROVED** [review]
Thank you!

File: cypress/support/pageObjects/LandingPage.ts [hunk 1]
```diff
@@ -12,22 +14,19 @@ export default class LandingPage {
...
-  private retryIfEarlyAccessNotLoaded(attempts: number = 3): void {
-    cy.url().then((currentUrl) => {
-      if (currentUrl !== this.earlyAccessUrl && attempts > 0) {
-        cy.get(this.earlyAccessPageButtonId).click();
-        cy.wait(500);
-        this.retryIfEarlyAccessNotLoaded(attempts - 1);
-      }
-    });
+  public getEmailErrorMessage(): Cypress.Chainable {
+    return cy.get(this.emailInputErrorId).should('be.visible');
```
Comment (ejwoods) [inline cypress/support/pageObjects/LandingPage.ts:22]: I noticed the assertion `should('be.visible')` was added to the get..ErrorMessage functions. Was this previously implemented elsewhere or just added for extra robustness?

**Review by ejwoods — APPROVED** [review]
Looks like a good migration to me 👍

File: cypress/support/pageObjects/LandingPage.ts (hunk 1 above)
Comment (martin-carmona) [reply cypress/support/pageObjects/LandingPage.ts:22]: Good catch, I added `.should('be.visible')` when I was debugging a bug where Cypress wasn't finding the error messages and left it for extra robustness.

**Pull request merged**